exporter/exporterhelper/internal/base_exporter.go

```diff
@@ -70,6 +70,10 @@ func NewBaseExporter(set exporter.Settings, signal pipeline.Signal, options ...O
 		}
 	}
 
+	if be.batcherCfg.MinSizeItems != nil || be.batcherCfg.MaxSizeItems != nil {
+		set.Logger.Warn("Using of deprecated fields `min_size_items` and `max_size_items`")
```

Comment:
Something like that?
```suggestion
		set.Logger.Warn("The exporter config fields `min_size_items` and `max_size_items` in the `batcher` section are deprecated in favor of `min_size` and `max_size`")
```